File: no 1/java/hardware.java

// Latihan 1

class Hardware extends Product{
	private String brand;
	private String model;

	// constructor
	Hardware(){
	}

	public void setBrand(String brand){
		this.brand = brand;
	}
	public String getBrand(){
		return this.brand;
	}
	
	public void setModel(String model){
		this.model = model;
	}
	public String getModel(){
		return this.model;
	}
}

File: no 2/python/hardware.py
class Hardware:

# kelas yang digunakan untuk mengimplementasikan kelas product

	def __init__(self):
	# konstruktor
		self.brand = ""
		self.model = ""

	def setBrand(self, brand):
	# set nilai atribut brand
		self.brand = brand;

	def getBrand(self):
	# mengembalikan nilai atribut brand
		return self.brand

	def setModel(self, model):
	# set nilai atribut model
		self.model = model
	def getModel(self):
	# mengembalikan nilai atribut model
		return self.model